constructor/osxpkg.py

```diff
@@ -119,8 +122,8 @@ def modify_xml(xml_path, info):
     else:
         welcome_path = None
         if info.get("welcome_file", "").endswith(".nsi"):
-            print(f"Warning: NSI welcome_file, {info['welcome_file'].endswith('.nsi')}, "
-                  "is ignored.")
+            logger.info(f"Warning: NSI welcome_file, {info['welcome_file'].endswith('.nsi')}, "
```

Comment:
```suggestion
            logger.info(f"Warning: NSI welcome_file, {info['welcome_file']}, "
```

Besides this typo, this should be %-formatted, in line with the other logging calls.

Reply:
Updated the PR description with some reasoning on this topic.